tests/test_packages/test_skills/test_erc1155.py

```diff
@@ -19,11 +19,18 @@
 
 """This test module contains the integration test for the generic buyer and seller skills."""
 
-import time
+# import pytest
 
 from aea.test_tools.decorators import skip_test_ci
 from aea.test_tools.test_cases import AEATestCaseMany, UseOef
 
+FUNDED_ETH_PRIVATE_KEY_1 = (
```

Comment:
we could add this as a pytest fixture instead, like we do for `oef_addr` and `oef_port`

Reply:
so we have much more control over the tests that depend on it

Reply:
Yes one day, since this is only one test atm which requires it I prefer to save myself that time.